### DISCUSSION TITLE
Admin Panel Synthesis

### DISCUSSION THREAD

**Discussion by bobgarner** [twbs/bootstrap#35959] [Show & Tell]
I recently went open source with a compiler that can synthesize a Spring Boot based microservice complete with an Admin Panel and I am using Bootstrap for the web interface. I love Bootstrap! I am not really a web developer but I was able to get something functional fairly quickly. As an example, I synthesized a web site that can host tutorials. It's Github repo can be found at: [ec-sb-tutorial-service](https://github.com/entityc/ec-sb-tutorial-service). 

For a short time I will have this tutorial website up and running if you want to check it out: http://ec-tutorial-j8-env.eba-x3q8m5p9.us-west-1.elasticbeanstalk.com. You will have to signup (find the link) - it asks for an email address but its not verified so you can use anything. This will give you access to the "student" part of the website, to be an "instructor" where you can see the admin panel, please register with a fake email (like `someone@example.com`) and place this email in a comment on this discussion. I will then make you an instructor so you can edit the example tutorials that are in there - and how Bootstrap is used.

I'm really trying to find someone who is good at Bootstrap that would be willing to contribute to my open source project so it can synthesize better looking admin panels. Any help or suggestions would be greatly appreciated!

**Comment by m0dul3** [comment]
Done

**Reply by bobgarner** [reply]
Thanks, I just made you an Instructor, you should be able to modify tutorials.

**Reply by bobgarner** [reply]
Let me know if you have any questions!
